fwcfg64/power.c

```diff
@@ -120,13 +123,20 @@ NTSTATUS FwCfgEvtDevicePrepareHardware(IN WDFDEVICE Device,
                 "VMCoreInfo index is 0x%x", ctx->index);
     if (!ctx->index)
     {
-        return STATUS_BAD_DATA;
+        TraceEvents(TRACE_LEVEL_VERBOSE, DBG_PNP,
```

Comment:
Will the line 110 not fail?    
if (FWCfgCheckSig(ctx->ioBase) ||
        FWCfgCheckFeatures(ctx->ioBase, FW_CFG_VERSION_DMA) ||
        FWCfgCheckDma(ctx->ioBase))
    {
        return STATUS_BAD_DATA;
    }

Reply:
@YanVugenfirer  It will not fail if fw_cfg is peresent

Reply:
On modern QEMU it can't fail

Reply:
Some very old versions can't do "DMA"